src/commands/run-build.yml

```diff
@@ -1,13 +1,14 @@
 description: >
-  Run builds using the MATLAB build tool. Use this command to run the tasks in the plan returned
-  by a file named buildfile.m in the root of your repository. To use the run-build command, you
-  need MATLAB R2022b or a later release.
+  Run a build using the MATLAB build tool. Use this command to run the tasks specified in a file
+  named `buildfile.m` in the root of your repository. To use the `run-build` command, you need MATLAB
+  R2022b or a later release.
 
 parameters:
   tasks:
     description: >
-      Space-separated list of tasks to run. If not specified, the command runs the default tasks
-      in the plan returned by buildfile.m as well as all the tasks on which they depend.
+      Tasks to run, specified as a list of task names separated by spaces. If you do not specify
+      tasks, the command runs the default tasks in `buildfile.m` as well as all the tasks on which
+      they depend.
```

Comment:
Do we need to add something like this here? (We have a similar statement for run-command.)

MATLAB exits with exit code 0 if the tasks run without error. Otherwise, MATLAB terminates with a nonzero exit code, which causes the command to fail.